sdks/python/apache_beam/transforms/window.py

```diff
@@ -55,6 +55,8 @@
 from typing import Iterable
 from typing import List
 from typing import Optional
+from typing import Generic
```

Comment:
the Generic type import should be after the `Any`, I'd make a suggestion for it but the UI doesn't want me doing multiline suggestions outside of lines you've touched in the PR